pubsub/unit_tests/test__gax.py

```diff
@@ -953,13 +951,12 @@ def mock_insecure_channel(host):
             return mock_channel
 
         host = 'CURR_HOST:1234'
-        connection = _Connection(in_emulator=True, host=host)
         patch = mock.patch.multiple(
             'google.cloud.pubsub._gax',
             PublisherClient=mock_publisher_api,
             insecure_channel=mock_insecure_channel)
         with patch:
-            result = self._call_fut(connection)
+            result = self._call_fut(None, host=host, secure=False)
```

Comment:
I'm fine doing that, though it makes the calling code a bit "worse":

```python
make(self.creds, host=self.conn.host, secure=not self.conn.in_emul)
```

becomes

```python
kwargs = {'host': self.conn.host}
if not self.conn.in_emul:
    kwargs['creds'] = self.creds
make(**kwargs)
```